rasa/core/domain.py

```diff
@@ -477,20 +478,47 @@ def get_parsing_states(self, tracker: "DialogueStateTracker") -> Dict[Text, floa
                         slot_id = "slot_{}_{}".format(key, i)
                         state_dict[slot_id] = slot_value
 
-        latest_message = tracker.latest_message
-
         if "intent_ranking" in latest_message.parse_data:
             for intent in latest_message.parse_data["intent_ranking"]:
                 if intent.get("name"):
                     intent_id = "intent_{}".format(intent["name"])
                     state_dict[intent_id] = intent["confidence"]
 
-        elif latest_message.intent.get("name"):
+        elif intent_name:
             intent_id = "intent_{}".format(latest_message.intent["name"])
             state_dict[intent_id] = latest_message.intent.get("confidence", 1.0)
 
         return state_dict
 
+    def _get_featurized_entities(
+        self, latest_message: Optional[UserUttered]
+    ) -> Set[Text]:
+        intent_name = latest_message.intent.get("name")
+        intent_config = self.intent_config(intent_name)
+        entities = latest_message.entities
+        entity_names = {
+            entity["entity"] for entity in entities if "entity" in entity.keys()
+        }
+
+        # `use_entities` is either a list of explicitly included entities
+        # or `True` if all should be included
+        include = intent_config.get("use_entities")
```

Comment:
I think this should be `True` by default @mauricedoepke 
```suggestion
        include = intent_config.get("use_entities", True)
```